
Removes git commits to vulndb since it seems to be broken
Tracks vulndb repository in /tmp/vulndb_data

--- vulndb-0.0.19.orig/tools/update-db.sh	2015-11-17 22:47:34.506772912 -0700
+++ vulndb-0.0.19/tools/update-db.sh	2015-11-17 22:54:12.818745899 -0700
@@ -5,25 +5,13 @@
 #
 # Update the database
 #
-if [ ! -d "data" ]; then
-  git clone https://github.com/vulndb/data.git
+
+if [ ! -d "/tmp/vulndb_data" ]; then
+  git clone https://github.com/vulndb/data.git /tmp/vulndb_data
 fi
 
-cd data
+cd /tmp/vulndb_data
 git pull
 cd ..
 
-mkdir -p vulndb/db/
-cp -rf data/db/*.json vulndb/db/
-git add vulndb/db/
-
-# Bump the version numbers
-tools/semver.sh bump patch
-
-cd data
-git rev-parse HEAD > ../vulndb/db-version.txt
-cd ..
-
-# Push to repo
-git commit vulndb/db/ vulndb/version.txt vulndb/db-version.txt -m 'Updated vulnerability database'
-git push
\ No newline at end of file
+cp -rf /tmp/vulndb_data/db/*.json /usr/lib64/python2.7/site-packages/vulndb/db/
